client/src/templates/Challenges/classic/lower-jaw.tsx

```diff
@@ -149,7 +149,7 @@ const LowerJaw = ({
               <p>{t(sentencePicker())}</p>
             </div>
           </div>
-          <div className='hint-status fade-in' aria-hidden={isFeedbackHidden}>
+          <div className='hint-status' aria-hidden={isFeedbackHidden}>
```

Comment:
```suggestion
          <div className='hint-status fade-in' aria-hidden={isFeedbackHidden}>

Reply:
I realized while messing with that class, I had reduced motion on the whole time